django/forms/widgets.py

```diff
@@ -851,10 +851,16 @@ class SplitDateTimeWidget(MultiWidget):
     supports_microseconds = False
     template_name = 'django/forms/widgets/splitdatetime.html'
 
-    def __init__(self, attrs=None, date_format=None, time_format=None):
+    def __init__(self, attrs=None, date_format=None, time_format=None, date_attrs=None, time_attrs=None):
         widgets = (
-            DateInput(attrs=attrs, format=date_format),
-            TimeInput(attrs=attrs, format=time_format),
+            DateInput(
+                attrs=attrs if date_attrs is None else date_attrs,
```

Comment:
Thanks again. I added tests for this logic. I also changed [one line](https://github.com/django/django/pull/7843/files#diff-aee6730bae795d31efec3c5d014804a9L859) in `django/forms/widgets.py`, because if we pass `attrs` to `MultiWidget` then it always overrides subwidgets `attrs` in `get_context()` (see [widgets.py#L807](https://github.com/django/django/blob/master/django/forms/widgets.py#L807)).